docs-md/customization/exemptions.md

```diff
@@ -18,19 +21,25 @@ kubectl annotate deployment my-deployment polaris.fairwinds.com/cpuRequestsMissi
 
 ## Config
 
-To exempt a controller via the config, you have to specify a namespace (optional), a list of controller names and a list of rules, e.g.
+You can add exemptions by using a combination of namespace, controller names, and container names via the config. You have to specify a list of rules and at least one of the following: a namespace, a list of controller names, or a list of container names, e.g.
 ```yaml
 exemptions:
-  # exemption valid for kube-system namespace
+  # exemption valid in kube-system namespace and dns-controller controller for all containers
   - namespace: kube-system
     controllerNames:
       - dns-controller
     rules:
       - hostNetworkSet
-  # exemption valid in all namespaces
+  # exemption valid in all namespaces and dns-controller controller for all containers
   - controllerNames:
       - dns-controller
     rules:
       - hostNetworkSet
+  # exemption valid in kube-system namespace and all controllers for coredns container
```

Comment:
We should probably add one that's a particular namespace but all rules and all controllers and all containers.

Reply:
Good point! I updated the documentation by borrowing some from #457 PR